libnl-route-3.sym

```diff
@@ -1331,7 +1331,6 @@ global:
 	rtnl_route_get_nhid;
 	rtnl_route_nh_identical;
 	rtnl_route_set_nhid;
-	rtnl_link_get_perm_addr;
```

Comment:
This commit fixes a problem if the previous commit, within the same PR.

In this case, could you please instead squash the commits, so that there is only commit. The important part is not that there is only one commit, but that each commit is self-contained as correct (as far as we know).

Also, could you please rebase to latest `main`? There will be a conflict, but it should be easy to resolve.

Reply:
done